src/helpers/buildWalletSections.tsx

```diff
@@ -105,23 +105,14 @@ const withPositionsSection = (isLoadingUserAssets: boolean) => {
 const withBriefBalanceSection = (
   sortedAssets: ParsedAddressAsset[],
   isLoadingUserAssets: boolean,
+  accountBalance: number | undefined,
```

Comment:
haven't confirmed but leaving a note to self: check `number` is alright here vs `string` for a formatted balance value